magento/inventory
# Check product saleable quantity before showing "only X left" message

Fixes #2042

As @krzksz already mentioned, we prepared a fix for this issue.

### Fixed Issues (if relevant)
1. magento-engcom/msi#2042: "Only X left" message displays for out of stock products

### Manual testing scenarios
1. Magento 2.3.0 with MSI.
2. `Only X left Threshold` option set to default `0`.
3. Simple product created with `0` quantity and `Out of Stock` status.
4. Go to product's page.
5. Check product's buybox.

### Contribution checklist
